src/chain.h

```diff
@@ -126,8 +126,8 @@ enum BlockStatus: uint32_t {
     //! Unused.
     BLOCK_VALID_UNKNOWN      =    0,
 
-    //! Parsed, version ok, hash satisfies claimed PoW, 1 <= vtx count <= max, timestamp not in future
-    BLOCK_VALID_HEADER       =    1,
+    //! Unused (was BLOCK_VALID_HEADER).
```

Comment:
Can you say "reserved" instead of "unused" here? Also in the commit description and the PR description? Sorry for being so literal, but if I see a comment telling me that a constant is unused, I really expect it to be unused in the sense that changing it to a different value would have no effect.

Reply:
Done